metadata-io/src/test/java/com/linkedin/metadata/graph/neo4j/Neo4jGraphServiceTest.java

```diff
@@ -160,4 +170,24 @@ public void testConcurrentRemoveNodes() {
     throw new SkipException("Neo4jGraphService produces duplicates");
   }
 
+  @Test
+  public void testRemoveEdge() throws Exception {
+    DatasetUrn datasetUrn = new DatasetUrn(new DataPlatformUrn("snowflake"), "test", FabricType.TEST);
+    TagUrn tagUrn = new TagUrn("newTag");
+    Edge edge = new Edge(datasetUrn, tagUrn, TAG_RELATIONSHIP, null, null, null, null, null);
+    getGraphService().addEdge(edge);
+
+    RelatedEntitiesResult result = getGraphService().findRelatedEntities(Collections.singletonList(datasetType),
+        newFilter(Collections.singletonMap("urn", datasetUrn.toString())), Collections.singletonList("tag"),
+        EMPTY_FILTER, Collections.singletonList(TAG_RELATIONSHIP),
+        newRelationshipFilter(EMPTY_FILTER, RelationshipDirection.OUTGOING), 0, 100);
+    assertEquals(result.getTotal(), 1);
+    getGraphService().removeEdge(edge);
+
+    result = getGraphService().findRelatedEntities(Collections.singletonList(datasetType),
+        newFilter(Collections.singletonMap("urn", datasetUrn.toString())), Collections.singletonList("tag"),
+        EMPTY_FILTER, Collections.singletonList(TAG_RELATIONSHIP),
+        newRelationshipFilter(EMPTY_FILTER, RelationshipDirection.OUTGOING), 0, 100);
+    assertEquals(result.getTotal(), 0);
+  }
```

Comment:
Would it be possible to add more tests to make sure the time filtering is working correctly?

Reply:
Otherwise, have you been able to verify that this behaves correctly? Screenshots would be helpful :)

Reply:
added testGetLineageTimeFilterQuery in Neo4jGraphServiceTest.java

Reply:
Thank you!